integration-tests/helpers.js

```diff
@@ -38,6 +41,69 @@ class FakeAgent extends EventEmitter {
         payload: msgpack.decode(req.body, { codec })
       })
     })
+    app.post('/v0.7/config', (req, res) => {
+      const buffers = []
+      req.on('data', buffers.push.bind(buffers))
+      req.on('end', () => {
+        const { products } = JSON.parse(Buffer.concat(buffers).toString()).client
+
+        const expires = (new Date(Date.now() + 1000 * 60 * 60 * 24)).toISOString() // in 24 hours
+        const clientID = uuid() // TODO: What is this? It isn't the runtime-id
+
+        // Currently, only `opaque_backend_state` and `targets` are used by dd-trace-js in the object below
```

Comment:
we use `opaque_backend_state` ??? this field should be as it's called, opaque, it's just a value that we need to keep in memory so the backend doesn't have to. Not implementing it shouldn't cause any problem ?

Reply:
Yes, it's referenced here:

https://github.com/DataDog/dd-trace-js/blob/2ded9362c2eac679791c55b6c7b5c14574362f07/packages/dd-trace/src/appsec/remote_config/manager.js#L218

Reply:
besides that, it's not being used for anything (e.g. it's not being parsed), so we could technically send down any random value or nothing at all and the code would continue to function.

Do you want me to make any changes?

Reply:
I'll remove it in an upcoming commit 👍

Reply:
Fixed 👍